### DISCUSSION TITLE
Save screenshot on mobile

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#196] [Feature Requests] [answered]
I'm pretty sure it used to, but, the "save screenshot" op isn't saving my image to the camera roll on ios mobile. 
Is this fixable?

here's the example patch for reference.
https://cables.gl/p/OB0Qmi

Thinking about it as well it would be great if we could access the width and height values. Seems possible to set but no ports.

**Comment by pandrr** [comment, marked as answer]
sadly not possible by default 
but now there is "web share api", which seems interesting...

https://web.dev/web-share/
https://caniuse.com/?search=Web%20Share%20API

fixed the ports, they are shown on dev.cables.gl now
